src/routing/routeConfiguration.js

```diff
@@ -134,6 +136,24 @@ const routeConfiguration = (layoutConfig, accessControlConfig) => {
       component: EditListingPage,
       loadData: pageDataLoadingAPI.EditListingPage.loadData,
     },
+    {
+      path: '/listing/new',
```

Comment:
Ain't this too general?

Let's discuss this some more, but I think it should be a bit more specific given that is ONLY for produts